tools/webhook-apicoverage/webhook/webhook.go

```diff
@@ -147,16 +151,37 @@ func (acw *APICoverageWebhook) registerWebhook(rules []admissionregistrationv1be
 	return nil
 }
 
+func (acw *APICoverageWebhook) getValidationRules(resources map[schema.GroupVersionKind]webhook.GenericCRD) ([]admissionregistrationv1beta1.RuleWithOperations) {
```

Comment:
this method doesn't consume acw at all, why it's part of acw?

Reply:
Its more of an encapsulation thing. Hence it is not exposed outside the package.